### ISSUE TITLE
Icons become extra huge after Obsidian v.1.0.0

### ISSUE THREAD

**Issue by GinoReiji** [aidenlx/obsidian-icon-shortcodes#59]
The the latest Obsidian update stops supporting the theme I was using so I ended up deleting it and going back to the default theme. Now all the icons I put in front of the heading become extremely huge for some reason. Emoji ones are ok, but the svg files just get so big it covers the entire page 😟

**Comment by p-laranjinha** [comment]
I believe the fix should be as simple as changing `var(--h@{level})` to `var(--h@{level}-size)` in [this line](https://github.com/aidenlx/obsidian-icon-shortcodes/blob/857673087c729d19d186c889c9895eb144b440a8/src/icon-packs/icon.less#L22).

Meanwhile, @GinoReiji, you could use this snippet:
```css
.markdown-source-view .HyperMD-header-1 .isc-icon>img,
.markdown-source-view .HyperMD-header-1 .isc-icon>svg,
.markdown-preview-view h1 .isc-icon>img,
.markdown-preview-view h1 .isc-icon>svg {
    height: var(--h1-size);
}

.markdown-source-view .HyperMD-header-2 .isc-icon>img,
.markdown-source-view .HyperMD-header-2 .isc-icon>svg,
.markdown-preview-view h2 .isc-icon>img,
.markdown-preview-view h2 .isc-icon>svg {
    height: var(--h2-size);
}

.markdown-source-view .HyperMD-header-3 .isc-icon>img,
.markdown-source-view .HyperMD-header-3 .isc-icon>svg,
.markdown-preview-view h3 .isc-icon>img,
.markdown-preview-view h3 .isc-icon>svg {
    height: var(--h3-size);
}

.markdown-source-view .HyperMD-header-4 .isc-icon>img,
.markdown-source-view .HyperMD-header-4 .isc-icon>svg,
.markdown-preview-view h4 .isc-icon>img,
.markdown-preview-view h4 .isc-icon>svg {
    height: var(--h4-size);
}

.markdown-source-view .HyperMD-header-5 .isc-icon>img,
.markdown-source-view .HyperMD-header-5 .isc-icon>svg,
.markdown-preview-view h5 .isc-icon>img,
.markdown-preview-view h5 .isc-icon>svg {
    height: var(--h5-size);
}

.markdown-source-view .HyperMD-header-6 .isc-icon>img,
.markdown-source-view .HyperMD-header-6 .isc-icon>svg,
.markdown-preview-view h6 .isc-icon>img,
.markdown-preview-view h6 .isc-icon>svg {
    height: var(--h6-size);
}
```

**Comment by GinoReiji** [comment]
@p-laranjinha It works, thank you so much!!!! Nearly every note I created has custom svg in the headings so I've been having a massive headache until now... Thank you thank you thank you ❤️❤️❤️❤️❤️

**Comment by aidenlx** [comment]
should be fixed in v0.9.7
